src/utils.js

```diff
@@ -1,6 +1,9 @@
 'use strict'
 
 function idToHex (id) {
+  if (id == null) {
```

Comment:
This is a matter of personal taste, but I would prefer using strict equality to avoid having to remember the falsey rules, even though it's more verbose.

`if (id === null || id === undefined) return`